### PR TITLE
[DF-351] UI: added the item 'Terms and Conditions' to Admin menu

### PR DESCRIPTION
**Pull request by wwebec**

### CODE REVIEW

File: src/main/webapp/app/layouts/navbar/navbar.component.html [hunk 1]
```diff
@@ -206,6 +206,16 @@
                                             <span
                                                 jhiTranslate="global.menu.blacklisted-agent">Blacklist</span>
                                         </a>
+                                        <a class="dropdown-item"
+                                           routerLink="blacklisted-agent"
```
Comment (varusakov) [inline src/main/webapp/app/layouts/navbar/navbar.component.html]: Change it to `terms-and-conditions` or something similar

File: src/main/webapp/app/layouts/navbar/navbar.component.html [hunk 2]
```diff
@@ -206,6 +206,16 @@
                                             <span
                                                 jhiTranslate="global.menu.blacklisted-agent">Blacklist</span>
                                         </a>
+                                        <a class="dropdown-item"
+                                           routerLink="terms-and-conditions"
+                                           *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_SYSADMIN']"
+                                           routerLinkActive="active"
+                                           (click)="collapseNavbar()">
+                                            <span class="fa fa-fw fa-ban"
```
[the remark this answers is not in the document]
Comment (cjpsycos) [reply src/main/webapp/app/layouts/navbar/navbar.component.html]: use gavel or balance-scale

File: src/main/webapp/i18n/en/global.json [hunk 3]
```diff
@@ -207,6 +207,7 @@
             "company-contact-template": "Contact Information",
             "dashboard": "Dashboard",
             "blacklisted-agent": "Blacklist",
+            "termsAndConditions": "AGB",
```
Comment (cjpsycos) [inline src/main/webapp/i18n/en/global.json]: This is german in english transaltions

File: src/main/webapp/i18n/de/global.json [hunk 4]
```diff
@@ -207,6 +207,7 @@
             "company-contact-template": "Kontaktangaben",
             "dashboard": "Dashboard",
             "blacklisted-agent": "Blacklist",
+            "termsAndConditions": "Terms And Conditions",
```
Comment (cjpsycos) [inline src/main/webapp/i18n/de/global.json]: This is english in german translations

**Pull request merged**